shell/platform/darwin/macos/framework/Headers/FlutterEngine.h

```diff
@@ -89,9 +91,40 @@ FLUTTER_DARWIN_EXPORT
  *
  * Setting this field from non-nil to a different non-nil FlutterViewController
  * is prohibited and will throw an assertion error.
+ *
+ * This method is deprecated. Querying view controllers should use
```

Comment:
Are properties referred as methods in Objective-C?

```suggestion
 * This property is deprecated. Querying view controllers should use
```

Reply:
A getter and for mutable properties, a setter method are synthesised, but properties should be referred to as properties in docs.